content/courses/onchain-development/local-setup.md

```diff
@@ -81,32 +105,58 @@ anchor test
 ```
 
 **The `anchor test` command should complete with no errors or warnings**.
-However you may encounter issues, and we'll fix them below:
 
-##### `package `solana-program v1.18.12` cannot be built because it requires rustc 1.75.0 or newer` error
+**However you may encounter issues, and we'll fix them below:**
 
+#### `package `solana-program
+
+v1.18.12` cannot be built because it requires rustc 1.75.0 or newer` error
+
+This error is due to incompatible versions of `solana-program` and `solana-cli`.
 Run `cargo add solana-program@"=1.18.x"`, where `x` matches your version of
 `solana-cli`. Then re-run `anchor test`.
 
-##### `Error: Unable to read keypair file`
+#### `Error: Unable to read keypair file`
 
 Add a keypair to `.config/solana/id.json`. You can either copy a keypair from an
 `.env` file (just the array of numbers) into a file or use the command
 `solana-keygen new --no-bip39-passphrase` to create a new keypair file. Then
 re-run `anchor test`.
 
-##### `unused variable: 'ctx'` warning
+#### error: no such command: `build-sbf`
+
+This error typically occurs because the Solana Build SBF (Solana BPF) feature is
+not available in your current environment. Solana BPF is used for compiling
+programs to run on the Solana blockchain. The error might indicate that either
+the Solana installation is corrupted or the relevant binaries are not in your
+system's PATH variable.
+
+One possible solution is to add the Solana binaries to your system's PATH
+variable. Add this to your `~/.zshrc` or `~/.bashrc` file to make the change
+permanent.
```

Comment:
```suggestion
If you see this message, this error typically occurs because the relevant binaries are not in your shell's PATH variable.

Run this command to add this folder to your shell, and also add this to your `~/.zshrc` or `~/.bashrc` file to make the change permanent.
```

Reply:
A few comments to help with future PRs (because your work is good and we want to do more in future):

 - 'system' is vague. Specifically, the thing we're fixing here is the shell. So let's say that!
 - 'Install is corrupted' is vague and kind of scary. How would an install be corrupted? The error is just that the shell needs a folder added to PATH. So let's be specific!